examples/MLIRLinalg/makefile

```diff
@@ -152,34 +156,97 @@ linalg-batch-matmul-optimize-run:
 		-shared-libs=${MLIR_RUNNER_UTILS} -shared-libs=${MLIR_C_RUNNER_UTILS}
 
 linalg-batch-matmul-lower:
-	@${MLIR_OPT} linalg-batch-matmul.mlir ${MLIR_OPT_OPTIONS} \
+	@${MLIR_OPT} linalg-batch-matmul_f32.mlir ${MLIR_OPT_OPTIONS} \
```

Comment:
Use `linalg-batch-matmul-f32` instead of `linalg-batch-matmul_f32` and modify the file name similarly.